docs/dev_guide/ceph_s3_tests/ceph_s3_tests_guide.md

```diff
@@ -137,70 +150,89 @@ Run the script that will create the necessary accounts in noobaa and update the
 ```
 
-Note: If you want to ignore PythonDeprecationWarnings use (which will then ignore all Python warnings, so keep that in mind):
+For example:
+
 ```bash
-export PYTHONWARNINGS="ignore"
+S3TEST_CONF=${PWD}/src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf tox -c src/test/system_tests/ceph_s3_tests/s3-tests/tox.ini -- --disable-pytest-warnings ${PWD}/src/test/system_tests/ceph_s3_tests/s3-tests/s3tests/functional/test_headers.py::test_bucket_create_contentlength_none
 ```
 
 ### 5) Run a Test (Inside The Tester Pod)
 To run a test, from noobaa working directory:
 ```bash
-S3TEST_CONF=src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf ./src/test/system_tests/ceph_s3_tests/s3-tests/virtualenv/bin/nosetests <test_name>
+S3TEST_CONF=${PWD}/src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf tox -c src/test/system_tests/ceph_s3_tests/s3-tests/tox.ini ${PWD}/src/test/system_tests/ceph_s3_tests/s3-tests/<test_name>
 ```
 This should run the test on the noobaa deployment we've set up.
 
 #### Test Name
-You can find a list of tests in the doc inside the file `ceph_s3_tests_list_single_test.txt`. Please notice that the test name has a certain structure <directory_name> are separated with `.` and the function to run (usually with a prefix `test_`) appears after the `:` sign.
+You can find a list of tests in the doc inside the file `ceph_s3_tests_list_single_test.txt`. Please notice that the test name has a certain structure: directories are separated with `/`, the files end with the extension `.py` and the function to run (usually with a prefix `test_`) appears after the `::` sign.
+
+In case the test name is incorrect, for example if you add `:` instead of `::` to the test name, the command will fail.
+The error will be `file or directory not found` and pytest will exit with error code 4 (which means "pytest command line usage error")
+
+#### Disable pytest warnings
+If you want to disable summary warnings add the following flag to the test command:
+`-- --disable-pytest-warnings`
+
+Note that every flag that comes after `--` is passed to pytest from tox. so if there is already `--` in the command just put `--disable-pytest-warnings` as part of the flags after it, no need to add another `--` notation.
+
+for example to add --disable-pytest-warnings to the command:
+```bash
+S3TEST_CONF=${PWD}/src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf tox -c src/test/system_tests/ceph_s3_tests/s3-tests/tox.ini -- -m 'not fails_on_rgw' ${PWD}/src/test/system_tests/ceph_s3_tests/s3-tests/s3tests_boto3/functional/test_s3.py::test_account_usage
+```
+
+it should be:
+```bash
+S3TEST_CONF=${PWD}/src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf tox -c src/test/system_tests/ceph_s3_tests/s3-tests/tox.ini -- -m 'not fails_on_rgw --disable-pytest-warnings ${PWD}/src/test/system_tests/ceph_s3_tests/s3-tests/s3tests_boto3/functional/test_s3.py::test_account_usage
```

Comment:
I would remove it (but leave the comment before) since we understood that we will not touch those flags when we run it.

Reply:
We still need an example even if the flag is not used. we don't have another flag that we do use, and that way it will be easy to replace the flag with a desired one when needed